tests/add.bats

```diff
@@ -108,35 +108,62 @@ load helpers
 @test "add single file creates absolute path with correct permissions" {
   imgName=ubuntu-image
   createrandom ${TESTDIR}/distutils.cfg
+  permission=$(stat -c "%a" ${TESTDIR}/distutils.cfg)
 
   cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ubuntu)
   run_buildah add $cid ${TESTDIR}/distutils.cfg /usr/lib/python3.7/distutils
   run_buildah run $cid stat -c "%a" /usr/lib/python3.7/distutils
   root=$(buildah mount $cid)
-  expect_output "644"
+  expect_output $permission
   run_buildah commit --signature-policy ${TESTSDIR}/policy.json $cid containers-storage:${imgName}
   run_buildah rm $cid
 
   newcid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ${imgName})
   run_buildah run $newcid stat -c "%a" /usr/lib/python3.7/distutils
-  expect_output "644"
+  expect_output $permission
   run_buildah rm $newcid
 }
 
 @test "add single file creates relative path with correct permissions" {
   imgName=ubuntu-image
   createrandom ${TESTDIR}/distutils.cfg
+  permission=$(stat -c "%a" ${TESTDIR}/distutils.cfg)
 
   cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ubuntu)
   run_buildah add $cid ${TESTDIR}/distutils.cfg lib/custom
   run_buildah run $cid stat -c "%a" lib/custom
   root=$(buildah mount $cid)
-  expect_output "644"
+  expect_output $permission
   run_buildah commit --signature-policy ${TESTSDIR}/policy.json $cid containers-storage:${imgName}
   run_buildah rm $cid
 
   newcid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ${imgName})
   run_buildah run $newcid stat -c "%a" lib/custom
-  expect_output "644"
+  expect_output $permission
   run_buildah rm $newcid
 }
+
+@test "add with chown" {
+  createrandom ${TESTDIR}/randomfile
+  cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json busybox)
+  run_buildah add --chown bin:bin $cid ${TESTDIR}/randomfile /tmp/random
+  run_buildah run $cid ls -l /tmp/random
+
+  expect_output --substring bin.*bin
+  buildah ps
+  buildah images
+  buildah rm $cid
+  buildah rmi busybox
+}
+
+@test "add url" {
+  cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json busybox)
+  run_buildah add $cid https://github.com/containers/buildah/raw/master/README.md
+  run_buildah run $cid ls /README.md
+
+  run_buildah add $cid https://github.com/containers/buildah/raw/master/README.md /home
+  run_buildah run $cid ls /home/README.md
```

Comment:
Please add `expect_output`. 

And, are both tests really necessary? If they are, e.g. because one specifies an explicit destination and the other assumes an implicit one, then please add a comment describing the reason.

Reply:
As run_buildah will raise en error if the file is not exist. So I thought the expect_output is not needed here. 
And for the test case itself, as long as it is supported, if we don't cover it there should be some code is not be tested. So I think it is worth for us to try to cover all flags and options in the test.